### PR TITLE
ScanConfirmationView programmatic layout added; methods naming fixed

### PR DESCRIPTION
**Pull request by mobileapps23**

### CODE REVIEW

File: Snabble/UI/Scanner/ScanConfirmationView.swift [hunk 1]
```diff
@@ -8,90 +8,229 @@ import UIKit
...
+        let productStack = UIStackView()
+        productStack.translatesAutoresizingMaskIntoConstraints = false
+        productStack.axis = .vertical
+        productStack.distribution = .fill
+        productStack.alignment = .center
+        productStack.spacing = 4
+
+        let subtitleLabel = customLabel
+        subtitleLabel.font = UIFont.systemFont(ofSize: 13)
+        subtitleLabel.text = "Brand"
```
Comment (mrgrauel) [inline Snabble/UI/Scanner/ScanConfirmationView.swift:97]: Why did you set the text? Shouldn't it be ok that it's empty?

File: Snabble/UI/Scanner/ScanConfirmationView.swift [hunk 2]
```diff
@@ -8,90 +8,229 @@ import UIKit
...
+
+        let productNameLabel = customLabel
+        productNameLabel.font = UIFont.systemFont(ofSize: 17, weight: .bold)
+        productNameLabel.text = "Name"
```
Comment (mrgrauel) [inline Snabble/UI/Scanner/ScanConfirmationView.swift:101]: Why did you set the text? Shouldn't it be ok that it's empty?

File: Snabble/UI/Scanner/ScanConfirmationView.swift [hunk 3]
```diff
@@ -8,90 +8,229 @@ import UIKit
...
+
+        let originalPriceLabel = customLabel
+        originalPriceLabel.font = UIFont.systemFont(ofSize: 17)
+        originalPriceLabel.textColor = .secondaryLabel
+        originalPriceLabel.text = "Orig Price"
```
Comment (mrgrauel) [inline Snabble/UI/Scanner/ScanConfirmationView.swift:106]: Why did you set the text? Shouldn't it be ok that it's empty?

File: Snabble/UI/Scanner/ScanConfirmationView.swift [hunk 4]
```diff
@@ -8,90 +8,229 @@ import UIKit
...
+
+        let priceLabel = customLabel
+        priceLabel.font = UIFont.monospacedDigitSystemFont(ofSize: 17, weight: .regular)
+        priceLabel.text = "Price"
```
Comment (mrgrauel) [inline Snabble/UI/Scanner/ScanConfirmationView.swift:110]: Why did you set the text? Shouldn't it be ok that it's empty?

**Review by mrgrauel — CHANGES_REQUESTED** [review]

**Comment by mobileapps23** [comment]
Label's placeholders was removed.

**Review by mrgrauel — APPROVED** [review]

**Pull request merged**